core/lexicon/en/menu.inc.php

```diff
@@ -16,7 +16,7 @@
 $_lang['lexicon_key'] = 'Lexicon Key';
 $_lang['lexicon_key_desc'] = 'The text, or lexicon key, that will be used for rendering the title text of this page in the menu.';
 $_lang['menu_create'] = 'New Menu';
-$_lang['menu_confirm_remove'] = 'Are you sure you want to remove this menu?';
+$_lang['menu_confirm_remove'] = 'All nested menus also will be deleted. Are you sure you want to remove this menu? ';
```

Comment:
To keep it consistent with the lexicon [`resource_delete_confirm`](https://github.com/modxcms/revolution/blob/cfa09fabf3914d6012f7c4606cf1e9e601bd95a4/core/lexicon/en/resource.inc.php#L47) I propose to use this lexicon instead:
```suggestion
$_lang['menu_confirm_remove'] = 'Are you sure you want to remove this menu?<br />NOTE: Any nested menus will also be deleted!';
```

Reply:
Then it is better to write "to delete" instead of "to remove", as in the resource lexicon.